### DISCUSSION TITLE
Powershell installation, how to?

### DISCUSSION THREAD

**Discussion by MichiFr** [starship/starship#2043] [Q&A] [answered]
Hi,

Unfortunately the installation instructions do not mention how to install the necessary files for PowerShell. I do not use Chocolatey, but have successfully installed Starship into Linux.

Looking fo any help and maybe the missing steps can be added to the Readme? 

Thanks, 
Michael

**Comment by karlosos** [comment, marked as answer]
I have installed it with scoop 

```
scoop install starship
```

as is described in [docs](https://github.com/starship/starship#with-scoop).

Then I've added to my powershell `$PROFILE` script following line: 

```
Invoke-Expression (&starship init powershell)
```

as described [here](https://github.com/starship/starship#powershell).

To check path to `$PROFILE` file you can do `echo $PROFILE` or edit file with `code $PROFILE` or `vim $PROFILE`.
